.github/pull_request_template.md

```diff
@@ -1,18 +1,5 @@
 ### Description
 
-### Dragons
```

Comment:
I use dragons pretty frequently. I would prefer to keep a section for callouts to the reviewer